challenge-sizebay/src/utils/getDayOfTheWeek.jsx

```diff
@@ -0,0 +1,18 @@
+export const getDayOfTheWeek = (number) => {
+  switch (number) {
+    case 0:
+      return "Sunday";
+    case 1:
+      return "Monday";
+    case 2:
+      return "Tuesday";
+    case 3:
+      return "Wednesday";
+    case 4:
+      return "Thursday";
+    case 5:
+      return "Friday";
+    default:
+      return "Saturday";
+  }
```

Comment:
ref: this goes the same way as described above, but it's even simpler! You don't even need to pass an argument to your function, you can let the Date API do that for you instead.

```js
/**
 * Hey Luiz Fernando, check this out!
 * Get the current day of the week
 * @returns
*/
function getCurrentDayOfTheWeek() {
  const date = new Date();
  return date.toLocaleString('en-us', { weekday: 'long' });
}
```

Reply:
Done